dashboard/app/models/level.rb

```diff
@@ -216,7 +216,9 @@ def delete_custom_level_file
     ['Unplugged', # no solutions
      'TextMatch', 'Multi', 'External', 'Match', 'ContractMatch', 'LevelGroup', # dsl defined, covered in dsl
      'Applab', 'Gamelab', # all applab and gamelab are freeplay
-     'NetSim', 'Odometer', 'Vigenere', 'FrequencyAnalysis', 'TextCompression', 'Pixelation'] # widgets
+     'EvaluationQuestion', # plc evaluation
+     'NetSim', 'Odometer', 'Vigenere', 'FrequencyAnalysis', 'TextCompression', 'Pixelation',
+    ] # widgets
```

Comment:
Anyway, I'm not inclined to factor this out with this change in order to keep the scope limited. But agreed that this logic should exist as a method defined by subclasses